CMakeLists.txt

```diff
@@ -206,7 +207,7 @@ message(STATUS "Compiling with support for: ${apilist}")
 # PkgConfig file
 string(REPLACE ";" " " req "${PKGCONFIG_REQUIRES}")
 string(REPLACE ";" " " api "${API_DEFS}")
-configure_file("rtmidi.pc.in" "rtmidi.pc" @ONLY)
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/rtmidi.pc.in" "rtmidi.pc" @ONLY)
```

Comment:
Note, I just realized after merging: The change on this line was accidental, but it is harmless I think. Sorry

Reply:
I'm not sure if it's necessary but I saw it and it seems correct.  I suppose the 'current directory' is changed when descending directories, hence your comment?  In any case, harmless, as you say.  I'll leave it.

Reply:
I believe when you do the add_subdirectory, path roots (including both relatively specified files like this one, and the CMAKE_CURRENT_SOURCE_DIR variable) reset to the new subdirectory within the scope of that file. So this line change should have literally no effect I think